tests/test_ops.py

```diff
@@ -46,9 +46,22 @@ def my_custom_fun(arr1, arr2, arr3):
 
 
 @pytest.mark.parametrize(
-    "meth_name", ["__pos__", "__neg__", "__abs__"],
+    "meth_name", ["log", "log10", "log1p", "log2", "sqrt", "sign", "round", "exp", "expm1", "floor", "ceil"],
```

Comment:
I'd test all the introduced methods

Reply:
`meth_name` is no longer accurate as these are functions.

Reply:
Done! All new methods are tested and placed in `parametrize`.